# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/ContentParent.h
# Commit: a902acfbc868
# Full Hash: a902acfbc8680811e4538f8d27e78e4fe6ca55f2
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2019-05-23 15:55:11
# Regressor Bug: 1525720
# File Overlap Count: 1
# Description:
#   Bug 1525720, part 13 - Stop inheriting nsIRemoteTab interface in BrowserParent. r=nika
#   
#   This commit removes nsIRemoteTab as a parent class from BrowserParent,
#   so that BrowserHost is the only concrete implementation of nsIRemoteTab.
#   
# ==============================================================================

diff -r 50475885250c -r a902acfbc868 dom/ipc/ContentParent.h
--- a/dom/ipc/ContentParent.h	Wed May 08 11:45:23 2019 -0500
+++ b/dom/ipc/ContentParent.h	Wed May 08 14:34:47 2019 -0500
@@ -681,7 +681,7 @@
       const bool& aSizeSpecified, nsIURI* aURIToLoad,
       const nsCString& aFeatures, const float& aFullZoom,
       uint64_t aNextRemoteTabId, const nsString& aName, nsresult& aResult,
-      nsCOMPtr<nsIRemoteTab>& aNewBrowserParent, bool* aWindowIsNew,
+      nsCOMPtr<nsIRemoteTab>& aNewRemoteTab, bool* aWindowIsNew,
       int32_t& aOpenLocation, nsIPrincipal* aTriggeringPrincipal,
       nsIReferrerInfo* aReferrerInfo, bool aLoadUri,
       nsIContentSecurityPolicy* aCsp);